src/js/game/components/goal_acceptor.js

```diff
@@ -31,19 +31,26 @@ export class GoalAcceptorComponent extends Component {
 
     clear() {
         // the last items we delivered
-        /** @type {{ item: BaseItem; time: number; }[]} */
-        this.deliveryHistory = [];
+        /** @type {{ item: BaseItem; time: number; }} */
+        this.lastDelivery = null;
+
+        this.currentDeliveredItems = 0;

```

Comment:
Why not use `this.clearItems()` here?

Reply:
Because that also clears the animation completely

Reply:
but isn't `clear()` the one that clears the animation?